test/box-luatest/gh_6144_incorrect_timeout_error_test.lua

```diff
@@ -0,0 +1,37 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_all = function()
+    g.server = server:new{
+        alias   = 'default',
+    }
+    g.server:start()
+end
+
+g.test_incorrect_timeout_error = function()
+    g.server:exec(function()
+        local t = require('luatest')
+        local net = require('net.box')
+        local s = box.schema.space.create('test')
+        s:create_index('primary', {type = 'TREE', parts = {1,'unsigned'}})
+        s:insert{78, 2, "string"}
+        box.schema.user.grant('guest', 'read,write', 'space', 'test')
+        local conn = net:connect(box.cfg.listen)
+        t.assert_error_msg_equals("timed out", function() conn.space.test:insert({1, "some string"}, {timeout = 0}) end)
```

Comment:
The line's too long. Please keep all code lines within 80 characters.

Anyway, I don't think we need this test, because the timeout message is checked in a more extensive way in a bunch of tests you patch. Let's drop it.

Reply:
Fixed